cardano-lib/testnet-template/config-legacy.json

```diff
@@ -76,7 +76,7 @@
   ],
   "hasEKG": 12788,
   "hasPrometheus": [
-    "0.0.0.0",
+    "127.0.0.1",
```

Comment:
Why this change?

Reply:
For the standard networks, metrics are exported by default to localhost so as to not expose them to the world if a suitable firewall isn't already in place.  This testnet config was inconsistent with the other cfgs and is now made consistent with them.  Arguably this is less important for a testnet template config, but overall still seemed reasonable to make consistent in approach.